src/components/player.js

```diff
@@ -250,13 +250,23 @@ export default class Player extends BaseObject {
     this.core.mediaControl.container.stop();
   }
 
+  
+  /**
+   * seeks the current video (`source`). For example, `player.seek(120)` will seek to second 120 (2minutes) of the current video.
+   * @method seekSeconds
+   * @param {Number} time should be a number between 0 and the video duration.
+   */
+  seek(time) {
+    this.core.mediaControl.container.setCurrentTime(time, true);
+  }
+
   /**
    * seeks the current video (`source`). For example, `player.seek(50)` will seek to the middle of the current video.
    * @method seek
    * @param {Number} time should be a number between 0 and 100.
    */
-  seek(time) {
-    this.core.mediaControl.container.setCurrentTime(time);
+  seekPercentage(percentage) {
+    this.core.mediaControl.container.setCurrentTime(percentage);
```

Comment:
Since we're going to dispatch two kinds of values on `CONTAINER_SEEK` event, maybe we need to redesign it:
```javascript
this.trigger(Events.CONTAINER_SEEK, {percentage: true, value: percentage}, this.name)
this.trigger(Events.CONTAINER_SEEK, {percentage: false, value: time}, this.name)
//or
this.trigger(Events.CONTAINER_SEEK_PERCENTAGE, percentage, this.name)
this.trigger(Events.CONTAINER_SEEK, time, this.name)
```

I'm in favor of the second (adding another event)